main/params.go

```diff
@@ -124,6 +124,8 @@ func init() {
 	// HTTP Server:
 	httpHost := fs.String("http-host", "127.0.0.1", "Address of the HTTP server")
 	httpPort := fs.Uint("http-port", 9650, "Port of the HTTP server")
+	// when using PnP and you want your http service to listen on a different port.
+	externalHttpPort := fs.Uint("http-port-external", 9650, "External port of the HTTP server")
```

Comment:
nit: my linter is complaining that this should be `externalHTTPPort`

Reply:
golint seems fine with it, but renamed..